content/how-to/rackspace-fabric/fabric-ticketing/index.md

```diff
@@ -64,7 +64,23 @@ The user must be granted ticketing permissions for the ticket navigation items t
 <img width="257" alt="FT11" src="https://user-images.githubusercontent.com/98875667/156183533-ce5b707d-2d71-45ea-b626-2c5c1ec7c9e4.png">
 
 4. Filter: This filter option can filter your tickets using options such as Device, Created By, Accounts, and Date Range 
-5. Refresh List: When you click on Refresh List option, the most recently created ticket appears on the top of the ticket list. 
+5. Refresh List: When you click on Refresh List option, the most recently created ticket appears on the top of the ticket list.
+
+##### Export Tickets
+<img class="fig-img" src="/support/how-to/fabric-ticketing/export-tickets_01.png" alt="An expanded drop-down showing the export ticket and close ticket options">
```

Comment:
Need to change "Close Tickets" to "Close Selected". As we click on Actions button, we could see in the drop down list Export Tickets and Close Selected. So our nomenclature in document should match with UI.